run_ci.sh

```diff
@@ -78,15 +78,11 @@ sudo mount -o bind /opt/data/installer-cache /home/notstack/.cache/kni-install/l
 
 # Clone the project being tested, "dev-scripts" will have been cloned in the jenkins
 # job definition, for all others we do it here
-ORG=openshift-metal3
```

Comment:
I wonder if we can provide a fallback for BASE_REPO

Reply:
Do we need to?  If it's a variable always set by jenkins then we shouldn't set a default?

Reply:
At least when triggering a job manually you have to populate these things by hand. AFAIK there is no "recheck" here.

Reply:
Ya, we could leave this code in to set ORG if BASE_REPO isn't set, but it still wouldn't work if the fork name doesn't match the repository name.

Reply:
Yeah we have to manually trigger a job with the appropriate parameters, but I don't think defaulting BASE_REPO really helps much with that, since we need to populate other parameters anyway?